src/components/card/Card.jsx

```diff
@@ -58,27 +58,41 @@ class Card extends Component {
   }
 
   getAssignees() {
-    if (this.props.assignees.length > 2) {
+    if (this.props.assignees.length > 3) {
       const members = this.props.assignees
+      if (this.props.cardResponsible === {}) {
+        // orders member by alphabetical order
+        members.sort((a, b) => a.initials < b.initials)
+      } else {
+        // if responsible is in members, sets responsible to be the firs user
+        if (members.find(element => element.id === this.props.cardResponsible.id) === true) {
+          const indexOfResponsible = members.findIndex(this.props.cardResponsible)
+          members.splice(indexOfResponsible, 0)
+          // orders member by alphabetical order
+          members.sort((a, b) => a.initials < b.initials)
+          members.splice(0, 0, this.props.cardResponsible)
+        }
+      }
       const moreMembers = (
         <div>
-          {members.slice(2, members.length).map(assignee => (
-            <p key={this.props.id + assignee.id} className="assigneeInitials">
-              {assignee.initials.toUpperCase()}
-            </p>
-          ))}
+          {members.slice(2, members.length)
+            .map(assignee => (
+              <p key={this.props.id + assignee.id} className="assigneeInitials">
+                {assignee.initials.toUpperCase()}
```

Comment:
The `toUpperCase()` is necessary ? I think it's already in uppercase in db

Reply:
it is as the user wrote his name : 
- Theo Gauchoux => TG
- theo gauchoux => tg
- theo Gauchoux => tG

So I think it is necessary and provides a constant way to display. Espacially since we'll be using data from other Prello apps